woocommerce/woocommerce-admin-test-helper
# Disable WooCommerce emails

This PR adds the code to disable WooCommerce emails.

### Screenshots
![screenshot-one wordpress test-2021 08 12-14_16_57](https://user-images.githubusercontent.com/1314156/129240153-d4087969-c4fa-41c3-90b9-05a8e59b6bef.png)


### Detailed test instructions:

- Install and activate [WooCommerce Smooth Generator](https://github.com/woocommerce/wc-smooth-generator).
- Install and activate [WP Mail Logging](https://wordpress.org/plugins/wp-mail-logging/).
- Go to `Tools` > `WCA Test Helper` > `Tools`.
- Press the `Run` button that is in the `Disable WC emails` row.
- Open a new tab with WP Mail Log (URL `/wp-admin/tools.php?page=wpml_plugin_log`)
- Now add a few products, customers, and orders.
- No email should be sent while they are turned off.
- Turn on the WC emails again and verify that the emails are now registered in WP Mail Log.
